dev-lang/dhall/dhall-1.42.2.ebuild

```diff
@@ -20,10 +18,13 @@ KEYWORDS="~amd64"
 IUSE="+http +http-client-tls"
 REQUIRED_USE="http-client-tls? ( http )" # carry over between version bumps
 
-RDEPEND="
-	>=dev-haskell/aeson-1.0.0.0:=[profile?] <dev-haskell/aeson-2.2:=[profile?]
+PATCHES=(
+	"${FILESDIR}"/dhall-1.42.2-cabal-doctest.patch
```

Comment:
This is great. I also re-made this patch but I must have messed up somewhere because the doctests ended up eating all my memory! Yours doesn't do this.